<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkBSplineRegistrationViewControls</class>
 <widget class="QWidget" name="QmitkBSplineRegistrationViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>562</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>287</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkBSplineRegistrationViewControls</string>
  </property>
  <widget class="QGroupBox" name="m_OptimizerGroup">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>269</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Optimizer</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>275</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QComboBox" name="m_OptimizerSelector">
       <item>
        <property name="text">
         <string>LBFGSOptimizer</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Gradient Convergence Tolerance</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_GradConvTolerance">
       <property name="text">
        <string>0.00001</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Line Search Accuracy</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="m_LineSearchAccuracy">
       <property name="text">
        <string>0.9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Default Step Length</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="m_DefaultStepLength">
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Max # Function Evaluations</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="m_FunctionEvaluations">
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QCheckBox" name="m_TraceOn">
       <property name="text">
        <string>Print out information on every iteration step</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>271</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Transform</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Grid Nodes Inside Image</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_NumberOfGridNodes">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Deformation Field</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>218</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QCheckBox" name="m_SaveDeformFieldCheck">
       <property name="text">
        <string>Save Deformation Field</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="m_DeformationField"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="m_BrowseDeformationField">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="m_ApplyDeformationField">
       <property name="text">
        <string>Apply deformation field</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="m_PrintDeformField">
       <property name="text">
        <string>Print deformation field</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
